packages/core/src/repoHelpers.ts

```diff
@@ -0,0 +1,12 @@
+import { newid } from 'cevitxe-signal-client'
+import { Repo } from './Repo'
+
+/**
+ * only used in tests
```

Comment:
Maybe rename to `repoTestHelpers`

Reply:
👍 
9dac485113dca4bf08a00fc83875a74a0c66d863